packages/terra-search-field/CHANGELOG.md

```diff
@@ -21,6 +23,7 @@ Unreleased
 * Components updated to use `injectIntl` to interface with `react-intl's` `intl` context.
 * Update jest test snapshot
 * updated package.json test scripts
+* updated source and jest files for changes in form-input
```

Comment:
This line can be removed, we have it listed already under the `### Changed` section under the `Unreleased` section.